applications/IgaApplication/custom_elements/laplacian_IGA_element.h

```diff
@@ -0,0 +1,234 @@
+// KRATOS ___ ___  _  ___   __   ___ ___ ___ ___
+//       / __/ _ \| \| \ \ / /__|   \_ _| __| __|
+//      | (_| (_) | .` |\ V /___| |) | || _|| _|
+//       \___\___/|_|\_| \_/    |___/___|_| |_|  APPLICATION
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Nicolò Antonelli
+//
+
+#if !defined(KRATOS_LAPLACIAN_IGA_ELEMENT_H_INCLUDED )
+#define  KRATOS_LAPLACIAN_IGA_ELEMENT_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/ublas_interface.h"
+#include "includes/variables.h"
+
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+class LaplacianIGAElement : public Element
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Counted pointer of LaplacianIGAElement
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(LaplacianIGAElement);
+
+    typedef Element BaseType;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructor.
+    LaplacianIGAElement(
+        IndexType NewId,
+        GeometryType::Pointer pGeometry);
+
+    LaplacianIGAElement(
+        IndexType NewId,
+        GeometryType::Pointer pGeometry,
+        PropertiesType::Pointer pProperties);
+
+    /// Destructor.
+    virtual ~LaplacianIGAElement();
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    Element::Pointer Create(
+        IndexType NewId,
+        NodesArrayType const& ThisNodes,
+        PropertiesType::Pointer pProperties) const override;
+
+    Element::Pointer Create(
+        IndexType NewId,
+        GeometryType::Pointer pGeom,
+        PropertiesType::Pointer pProperties) const override;
+
+
+
+    void CalculateLocalSystem(MatrixType& rLeftHandSideMatrix, VectorType& rRightHandSideVector, const ProcessInfo& rCurrentProcessInfo) override;
```

Comment:
```suggestion
    void CalculateLocalSystem(
        MatrixType& rLeftHandSideMatrix,
        VectorType& rRightHandSideVector,
        const ProcessInfo& rCurrentProcessInfo) override;
```
Please follow the style guide. Same for the methods below.